package.json

```diff
@@ -26,10 +27,14 @@
     "@trpc/react-query": "^10.45.2",
     "@trpc/server": "^10.45.2",
     "autoprefixer": "^10.4.19",
+    "class-variance-authority": "^0.7.0",
+    "clsx": "^2.1.1",
```

Comment:
Not needed. It is installed automatically from cva

Reply:
Removed in 5d5ddc7d9fbf4410d79f73d65ad1bd91b54f079f.